### PR TITLE
Levels

### PR DESCRIPTION
**Pull request by Obleo33**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -19,13 +18,14 @@ var bottomTerrain = [];
 var topTerrain = [];
 var score = 0;
 var timer = 2000;
+var level = 1;
```
Comment (Obleo33) [inline lib/index.js:21]: Add a level variable

File: lib/index.js [hunk 2]
```diff
@@ -19,13 +18,14 @@ var bottomTerrain = [];
...
 
-function spawnSeal() {
+function spawnSeal(object) {
   if (spawnCount >= 0) {
     spawnCount--;
   } else {
     if (seals.length< 5) {
-      seals.push(new Seal({context: context,canvas: canvas,maxSpeed: 3,minSpeed:1}));
+      seals.push(new Seal({context: context, canvas: canvas, maxSpeed: object.maxSpeed, minSpeed: object.minSpeed}));
```
Comment (Obleo33) [inline lib/index.js:28]: min and max speed are now passed in

File: lib/index.js [hunk 3]
```diff
@@ -93,11 +100,11 @@ function drawTime(x,y) {
     context.fillText("TIME:"+ countDown(), x, y);
 }
 
-function clickToStart(x,y){
-  context.font = "40px 'Press Start 2P'";
-  context.fillStyle = "#000000";
-  context.textAlign = "center";
-  context.fillText("Click to start",x,y);
+function levelUp(x,y) {
```
Comment (Obleo33) [inline lib/index.js:103]: Level to screen

File: lib/index.js [hunk 4]
```diff
@@ -150,6 +169,10 @@ $('#game').on('click',function(){
       gameOver(canvas.width * 0.5, 200);
       drawScore(canvas.width * 0.5, 250,"center");
       clickToStart(canvas.width * 0.5, 400);
+      seals = [];
+      if (level<5){
+        level ++;
```
Comment (Obleo33) [inline lib/index.js:175]: Level will increase by 1 at end of each round

File: lib/index.js [hunk 5]
```diff
@@ -136,7 +143,19 @@ $('#game').on('click',function(){
...
+        spawnSeal({maxSpeed: 3, minSpeed: 1});
+      } else if (level === 2){
+        spawnSeal({maxSpeed: 4, minSpeed: 2});
+      } else if (level === 3){
+        spawnSeal({maxSpeed: 5, minSpeed: 3});
+      } else if (level === 4){
+        spawnSeal({maxSpeed: 6, minSpeed: 4});
+      } else {
+        spawnSeal({maxSpeed: 10, minSpeed: 8});
+      }
```
Comment (Obleo33) [inline lib/index.js:156]: check level count and adjust seal speed

**Comment by juliankyer** [comment]
Awesome-O.

**Pull request merged**